SPECS/thrift/thrift.signatures.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Signatures": {
```

Comment:
could you list these in alphabetical order?

Reply:
updated